features/support/paths.rb

```diff
@@ -262,6 +262,12 @@ def path_to(page_name)
     when /^the [mM]y account page$/
       '/my/account'
 
+    when /^the [aA]ccess [tT]oken page$/
+      '/my/access_token'
+
+    when /^the [sS]ettings page$/
```

Comment:
The point is: there is already a rule that matches the string "the settings page", I did not check what happens in that case.

I would propose something less ambiguous, e.g. `my settings page` or `user settings page`.